src/iptvsimple/data/EpgEntry.cpp

```diff
@@ -184,8 +184,8 @@ bool EpgEntry::UpdateFrom(const xml_node& channelNode, const std::string& id, in
   if (!GetAttributeValue(channelNode, "start", strStart) || !GetAttributeValue(channelNode, "stop", strStop))
     return false;
 
-  long long tmpStart = ParseDateTime(strStart);
-  long long tmpEnd = ParseDateTime(strStop);
+  long long tmpStart = ParseDateTime(strStart) + (86400 * 66); //2020-01-30
```

Comment:
2020-01-30 ?

Reply:
Ha, that's my EPG data hack, should never have been in there!!!